saleor/core/tasks.py

```diff
@@ -21,3 +24,16 @@ def delete_event_payloads_task():
     EventPayload.objects.filter(
         deliveries__isnull=True, created_at__lte=event_payload_delete_period
     ).delete()
+
+
+@app.task(
+    autoretry_for=(ClientError,),
```

Comment:
Should we auto-retry only on `ClientError`?
@korycins

Reply:
Do we support only S3 for storing images? It looks like we took an AWS error from the lower layer used in versatile.